data_access_log/admin.py

```diff
@@ -0,0 +1,29 @@
+import django
+from django.contrib import admin
+
+from .models import DataAccessLog
+
+
+@admin.register(DataAccessLog)
+class DataAccessLogAdmin(admin.ModelAdmin):
+    list_display = ('model', 'viewset', 'user', 'timestamp')
+    list_display_links = ('model', 'viewset', 'timestamp')
+
+    fields = ('model', 'viewset', 'timestamp', 'user', 'url', 'response')
+    readonly_fields = fields
+
+    def has_view_permission(self, request, obj=None):
+        return True
```

Comment:
please do as in has_change_permission below

Reply:
Not necessary. Versions before 2.1 will ignore it. I could add a comment to be more clear about it.

Reply:
I mean, it is visible by default on 2.1. It should only be visible to users with the right permission added. Maybe just don't include it, as has_change_permission is already handling it properly